backend/scrapers/classes/parsers/ellucianCatalogParser.js

```diff
@@ -79,7 +79,15 @@ class EllucianCatalogParser extends EllucianBaseParser.EllucianBaseParser {
       depData.maxCredits = creditsParsed.maxCredits;
       depData.minCredits = creditsParsed.minCredits;
     } else {
-      macros.log('warning, nothing matchied credits', url, text);
+      macros.log('warning, nothing matched credits', url, text);
+    }
+
+    const courseAttributes = this.parseStacysMom(text);
+
+    if (courseAttributes) {
+      depData.courseAttributes = courseAttributes;
+    } else {
+      macros.log('warning, nothing matched course attributes', url, text);
```

Comment:
Also lets remove logging of the text here, its quite a lot of output when running this code